app.py

```
@@ -660,5 +664,172 @@ def render_category_analysis(prod_df, kpi_prod_df, has_prod_kpi):
+        # ==============================================
+        # INSIGHTS POR CATEGORÍAS
+        # ==============================================
+        
+        st.subheader("🔍 Insights por Categorías")
+        
+        # Obtener filtros actuales del sidebar (si existen)
+        try:
+            # Intentar usar los filtros de la pestaña de categorías si existen
+            year_range_insights = st.session_state.get('category_years', (df['year'].max()-2, df['year'].max()))
+            selected_categories_insights = st.session_state.get('selected_categories', df['category'].unique()[:5])
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)